app/javascript/components/ingredients/ingredients-container.vue

```diff
@@ -227,8 +230,11 @@ export default {
     async editIngredient() {
       this.showingEdit = !this.showingEdit;
       try {
-        const res = await editIngredient(this.ingredientToEdit.id, this.$refs.editIngredientInfo.form);
-        this.updateIngredient(res);
+        var ingredientsInfo = this.$refs.editIngredientInfo.form;
+        ingredientsInfo.ingredient_measures_attributes = ingredientsInfo
+          .ingredient_measures_attributes.filter(unit => unit.name && unit.quantity);
+        const res = await editIngredient(this.ingredientToEdit.id, ingredientsInfo);
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'res' is assigned a value but never used                          no-unused-vars